bears/stylus/StylintBear.py

```diff
@@ -0,0 +1,206 @@
+class StylintBear:
+    """
+    Attempts to catch little mistakes (duplication of rules for instance) and
+    to enforce a code style guide. This is particularly important with Stylus,
+    which is unopinionated when it comes to syntax. Like Stylus itself, this
+    linter opts for flexibility over rigidity.
+    """
+
+    LANGUAGES = {'Stylus'}
+    REQUIREMENTS = {NpmRequirement('stylint', '1.5.9')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Formatting', 'Syntax', 'Redundancy'}
+    SEE_MORE = 'https://github.com/SimenB/stylint'
+
+    @staticmethod
+    def generate_config(filename, file,
+                        allow_block_keyword: bool=False,
+                        allow_brackets: str='never',
+                        allow_colons: str='always',
+                        allow_colors: str='always',
+                        allow_spaces_after_commas: str='always',
+                        allow_spaces_after_comments: str='always',
+                        allow_css_literal: str='never',
+                        max_selector_depth: bool=False,
+                        check_duplicates: bool=True,
+                        allow_efficient_properties: str='always',
+                        set_extend_preference: bool=False,
+                        check_global_duplicates: bool=False,
+                        set_indent_preference: bool=False,
+                        allow_leading_zero: str='never',
+                        set_max_errors: bool=False,
+                        set_max_warnings: bool=False,
+                        mixed_spaces_and_tabs: bool=False,
+                        set_naming_convention: bool=False,
+                        set_strict_naming_convention: bool = False,
+                        allow_none_keyword: str='never',
+                        check_no_important_keyword: bool=True,
+                        allow_spaces_inside_parens: bool=False,
+                        allow_placeholder: str='always',
+                        prefix_vars_with_dollar: str='always',
+                        allow_semicolons: str='never',
+                        sort_order: str='alphabetical',
+                        allow_stacked_properties: str='never',
+                        check_valid_property: bool=True,
+                        allow_zero_units: str='never',
+                        z_index_normalize: bool=False):
+        """
+        :param allow_block_keyword:
+            When ``always`` expect the ``@block`` keyword when defining block
+            variables. When ``never``, expect no ``@block`` keyword when
+            defining block variables. When ``false``, do not check either way.
+        :param allow_brackets:
+            When ``always``, expect ``{}`` when declaring a selector. When
+            ``never``, expect no brackets when declaring a selector.
+        :param allow_colons:
```

Comment:
This is not about documentation.  is `allow_colons` bi-state or tri-state.  If so, use the `bool` type with values `(True, False, None)` instead using a string type.